examples/opf/experiments/classification/base_category/description.py

```diff
@@ -187,9 +187,9 @@
             # is correct here as opposed to 'columns')
             'synPermConnected': 0.1,
 
-            'synPermActiveInc': 0.1,
+            'synPermActiveInc': 0.05,
 
-            'synPermInactiveDec': 0.01,
+            'synPermInactiveDec': 0.008,
```

Comment:
@tomsilver Yes, that makes sense. @breznak Unfortunately these are somewhat application dependent, but we could put better values in the constructor than what we have now.  Perhaps that should be a separate Issue - it might have a big impact for components that are not setting values explicitly.